app/views/layouts/application.slim

```diff
@@ -7,14 +7,17 @@ html
     = stylesheet_link_tag 'application', media: 'all'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
   body
+    p
+      = link_to 'new user by devise', new_user_registration_path
```

Comment:
we use 2 links, because we have 2 types of user registration:
1 - when we create user by using users_controller
2 - when we create user by devise
If you change it by [13310cb](https://github.com/DenisUA/Ruby-hotel/commit/13310cbd31dc99ed7e007cd224a089b359f75857) I will delete it